shotover-proxy/tests/cassandra_int_tests/basic_driver_tests.rs

```diff
@@ -983,13 +977,55 @@ APPLY BATCH;"#,
     }
 }
 
+fn prepared_statements(session: &Session) {
+    run_query(session, "CREATE KEYSPACE test_prepare_statements WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };");
+    run_query(
+        session,
+        "CREATE TABLE test_prepare_statements.table_1 (id int PRIMARY KEY, x int, name varchar);",
+    );
+
+    run_query(
+        session,
+        r#"BEGIN BATCH
+                INSERT INTO test_prepare_statements.table_1 (id, x, name) VALUES (1, 11, 'foo');
+                INSERT INTO test_prepare_statements.table_1 (id, x, name) VALUES (2, 12, 'bar');
+                INSERT INTO test_prepare_statements.table_1 (id, x, name) VALUES (3, 13, 'baz');
+            APPLY BATCH;"#,
+    );
+
+    let prepared = session
```

Comment:
could we do an insert prepared statement as well? Would be good to test both read and write paths on the prepared statement side of things